web/src/app/shared/services/notifications/notifications.service.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { inject, TestBed } from '@angular/core/testing';
+
+import { NotificationService } from './notification.service';
+
+describe('ErrorsService', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [NotificationService]
+    });
+  });
+
+  it('should be created', () => {
+    const service: NotificationService = TestBed.get(NotificationService);
+    expect(service).toBeTruthy();
+  });
+
+  it('should be created', inject([NotificationService], (service: NotificationService) => {
+    expect(service).toBeTruthy();
+  }));
```

Comment:
why are you testing two times? What is the difference between those two tests?

Reply:
this is most likely an artifact from the short time, where i had notification.service as a class from which error.service and alert.service extended.
intellij auto renaming most likely did this.
the difference is that the second variation is type-safe, whie the first one with Testbed.get is not
i removed the first test